pegasus/sites.v3/code.org/public/educate/curriculum/middle-school.md.erb

```diff
@@ -42,6 +42,15 @@ Computer Science Discoveries is an introductory course for 6 - 10th grade studen
 <br>
 <hr>
 
+# Creating Apps for Devices
+
+Every day, we use devices — computers, phones, and more — that help us accomplish tasks both big and small. When students learn to program these devices, foundational computer science comes alive! In this module, which spans roughly 6 weeks, students will learn the fundamentals of programming devices.
+
+[<button>Learn about Maker</button>](/maker)&nbsp;&nbsp;[<button>Explore the Creating Apps for Devices Module</button>](https://studio.code.org/s/devices)&nbsp;&nbsp;
+
+<br>
+<hr>
```

Comment:
Same nit as above –– this code is also part of the high school page. Is it likely that these will diverge or stay the same? If the same, I'd vote for a partial here, but it's not blocking

Reply:
Excellent idea, and done! (Did this for the other possible partial too.)

Reply:
Wahoo!!